### DISCUSSION TITLE
v0.4.4 - Minor improvements

### DISCUSSION THREAD

**Discussion by elcharitas** [ngyn-rs/ngyn#171] [Announcements]
## What's Changed
* feat(routes): allow `GET` to match `HEAD` requests by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/151
* chore(core): standardize ngyn responses by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/152
* chore(shared): add support for response data peeking by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/153
* chore(core): default platform engine to `hyper` by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/154
* feat(shared): support for state mutation by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/156
* chore: version bump `0.4.4` by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/158
* v0.4.4 - Minor improvements by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/157
* chore: release workflow improvements by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/159
* chore: release workflow improvements by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/161
* chore: add made-in-nigeria badge by @kaf-lamed-beyt in https://github.com/ngyn-rs/ngyn/pull/162
* chore(workflow): patch release process by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/163
* Release workflows by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/164
* chore: add package meta for release by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/165
* chore: add package meta for release by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/166
* chore: create `cargo-ngyn` crate by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/167
* chore: create `cargo-ngyn` crate by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/168
* chore: lockfile config for releases by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/169
* chore: lockfile config for releases by @elcharitas in https://github.com/ngyn-rs/ngyn/pull/170

## New Contributors
* @kaf-lamed-beyt made their first contribution in https://github.com/ngyn-rs/ngyn/pull/162

**Full Changelog**: https://github.com/ngyn-rs/ngyn/compare/ngyn@0.4.3...ngyn@0.4.4

<hr /><em>This discussion was created from the release <a href='https://github.com/ngyn-rs/ngyn/releases/tag/ngyn%400.4.4'>v0.4.4 - Minor improvements </a>.</em>

**Comment by kaf-lamed-beyt** [comment]
Hi @elcharitas, did you type this out yourself or it was auto-generated

**Reply by elcharitas** [reply]
It was auto-generated
